clang/lib/CIR/Dialect/Transforms/IdiomRecognizer.cpp

```diff
@@ -175,8 +150,16 @@ void IdiomRecognizerPass::recognizeCall(CallOp call) {
   if (raiseIteratorBeginEnd(call))
     return;
 
-  if (raiseStdFind(call))
-    return;
+  bool remark = opts.emitRemarkFoundCalls();
+
+  using StdFunctionsRecognizer =
+      std::tuple<StdRecognizer<3, StdFindOp, StdFuncID::Find>>;
+
+  std::apply(
+      [&](auto... recognizers) {
+        (decltype(recognizers)::raise(call, getContext(), remark), ...);
+      },
```

Comment:
A successful `raise` call would erase the call op which could crash later `raise` calls. Thus we should short-circuit here:

```suggestion
      [&](auto... recognizers) {
        (decltype(recognizers)::raise(call, getContext(), remark) || ...);
      },
```